### PR TITLE
Changed readme grammar

### PR DESCRIPTION
**Pull request by drewren25**
This is my first commit onto here so I just thought updating the readme would be fitting for now. This is just a change to the readme in a few spots where I thought the grammar or wording could be improved so no tests for this commit. The first change I made was to change "you're using" to "you're planning to use" because I thought this future tense fit better in the context of showing users how to get started with React. If they're just getting started, they're probably planning to use it and not already using it. The second change I made was from "This is a great place to get started." to "This is a great place to start when looking to contribute to React for the first time." I just thought the first one cut off abruptly so I added a bit more depth and wanted to mention "React" again at the end of the readme because I thought it sounded nice. 
If anyone's willing, I'm pretty new to contributing to opensource and doing actual outside of classroom code(I'm a sophomore in college), can someone explain tests to me? Before I typed this commit up there was autofilled text that mentioned things like "yarn" and how we should include our own tests, what is that exactly? Thank you!

### CODE REVIEW

**Comment by samuei** [comment]
Congrats on your first Pull Request!

>Before I typed this commit up there was autofilled text that mentioned things like "yarn" and how we should include our own tests, what is that exactly?

[Yarn ](https://classic.yarnpkg.com/en/) is a package manager, like [npm](https://www.npmjs.com/) or (in the Python world) [pip](https://pip.pypa.io/en/stable/). I believe npm is more widely used out in the world, but Facebook likes yarn a lot, so that's what they're using here.

>can someone explain tests to me?

[Unit tests](https://en.wikipedia.org/wiki/Unit_testing) are a big topic, so I'll encourage you to give it a bigger look, but Facebook tends to use [jest](https://jestjs.io/) for their projects (which makes sense, since Facebook created jest!).

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ ReactDOM.render(
 
 This example will render "Hello Taylor" into a container on the page.
 
-You'll notice that we used an HTML-like syntax; [we call it JSX](https://reactjs.org/docs/introducing-jsx.html). JSX is not required to use React, but it makes code more readable, and writing it feels like writing HTML. If you're using React as a `<script>` tag, read [this section](https://reactjs.org/docs/add-react-to-a-website.html#optional-try-react-with-jsx) on integrating JSX; otherwise, the [recommended JavaScript toolchains](https://reactjs.org/docs/create-a-new-react-app.html) handle it automatically.
+You'll notice that we used an HTML-like syntax; [we call it JSX](https://reactjs.org/docs/introducing-jsx.html). JSX is not required to use React, but it makes code more readable, and writing it feels like writing HTML. If you're planning to use React as a `<script>` tag, read [this section](https://reactjs.org/docs/add-react-to-a-website.html#optional-try-react-with-jsx) on integrating JSX; otherwise, the [recommended JavaScript toolchains](https://reactjs.org/docs/create-a-new-react-app.html) handle it automatically.
```
Comment (robertpiosik) [inline README.md:55]: This paragraph is more about getting started with JSX rather with React. "Getting started" page was mentioned earlier in the "Documentation" section so at this point you should be already familiar with it.

File: README.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ Read our [contributing guide](https://reactjs.org/contributing/how-to-contribute
 
 ### Good First Issues
 
-To help you get your feet wet and get you familiar with our contribution process, we have a list of [good first issues](https://github.com/facebook/react/labels/good%20first%20issue) that contain bugs which have a relatively limited scope. This is a great place to get started.
+To help you get your feet wet and get you familiar with our contribution process, we have a list of [good first issues](https://github.com/facebook/react/labels/good%20first%20issue) that contain bugs which have a relatively limited scope. This is a great place to start when looking to contribute to React for the first time.
```
Comment (robertpiosik) [inline README.md:71]: I think it unnecessarily repeats the beginning wording "get you familiar". Also text "This is a great place to get started." is more to me an encouragement to take action and the paragraph is short enough to not summarise what one just have read.

**Comment by eps1lon** [comment]
Since these are all subjective nits that don't seem to alter or clarify the original meaning, I'm going to close. But thanks for your suggestions.

**Pull request closed without merging**